docs/A11y-Testing-Cross-Device.md

```diff
@@ -0,0 +1,11 @@
+# Testing on Multiple Devices for an A11y Swarm
```

Comment:
Is "A11y swarm" a well understood term outside of the BBC? This will be a public document, so I think we should we use simpler terminology, e.g. accessibility instead of a11y?

Reply:
Accessibility is referred to as a11y outside of the BBC, in fact this page describes it's origin and specifically it's focus around digital accessibility: https://www.a11yproject.com/about/#what-does-the-term-a11y-mean 

I'll add a bit more info about what a swarm is and link out to that page.